Includes/subAppRouter.cpp

```diff
@@ -150,5 +240,7 @@ void SubAppRouter::handleButtonUp(const SDL_ControllerButtonEvent &event) {
     case SDL_CONTROLLER_BUTTON_START:
       app.onStartReleased();
       break;
+    default:outputLine("Ignoring invalid button ID %d\n", button);
```

Comment:
Please insert a line break between `default:` and what follows.

Reply:
Fixed, sorry!